src/astra_driver.cpp

```diff
@@ -484,8 +497,8 @@ void AstraDriver::imageConnectCb()
     if (!color_started)
     {
       if (enable_streaming_) {
-        ROS_INFO("Starting color stream.");
-        device_->startColorStream();
+        ROS_INFO("Starting color stream. Fake start!");
+        //device_->startColorStream();
```

Comment:
What is the purpose of this commented code ? enable_streaming_ does nothing.

Reply:
Do nothing.